react/features/google-api/constants.js

```diff
@@ -1,14 +1,23 @@
 // @flow
 
 /**
- * The Google API scopes to request access to for streaming.
+ * The Google API scopes to request access for streaming and calendar.
  *
  * @type {Array<string>}
  */
 export const GOOGLE_API_SCOPES = [
-    'https://www.googleapis.com/auth/youtube.readonly'
+    'https://www.googleapis.com/auth/youtube.readonly',
+    'https://www.googleapis.com/auth/calendar'
```

Comment:
Are there any changes needed to deployed applications to get calendar permissions?

Reply:
I think so, you need to just enable that API. We should take care of that about meet.jit.si. I was also thinking of dropping a simple README with links to Google and Microsoft dashboard how to enable these.